### ISSUE TITLE
Upload Pictures from Mobile not working

### ISSUE THREAD

**Issue by daqattack** [worbli/portal-v2-uat#5]
Reproduce and result: After instantly getting a message with the link on my iPhone, I hit the link but it shows a blank page. 

Expect result: I should get a page that allows me to upload an ID picture from my phone.

**Comment by bkawk** [comment]
I am able to view the page that asks me to enable the camera, as soon as I click the blue button to enable it the screen is blank. Is this the same as what you experience?

**Comment by daqattack** [comment]
No, I use an iPhone 6 and when I clicked on the link, it directed me to a page, but the page after loading, is blank so I could not see anything.
